### DISCUSSION TITLE
How can I login in with another Google account

### DISCUSSION THREAD

**Discussion by futantan** [nextauthjs/next-auth#3795] [Help] [answered]
### Question 💬

When I logged in with a google account, and then sign out, finally sign in again, it will automatically sign in with the previous account, I have no chance to choose which account I'd like to use.

### How to reproduce ☕️

https://next-auth-example.vercel.app/

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment, marked as answer]
This is not related to NextAuth.js, but how OAuth usually works by default. 

https://next-auth.js.org/getting-started/client#additional-parameters

https://www.oauth.com/oauth2-servers/authorization/requiring-user-login/#:~:text=In%20Google%E2%80%99s%20API,prompt%3Dlogin

**Comment by damain** [comment]
I've read this answer before and didn't understand it and I know of other teams that switched from using next-auth because they can't solve this issue. 

The answer to this is to prompt the user to login.(As stated in the documentation)

What I did was to create a component that is displayed if you are not logged in. On this component I have a button that calls the signIn function from 'next-auth/react'. Here I am using Azure as my OAuth provider. You can swap out "azure-ad"  for "google" if Google is your provider. The `{ prompt: "login" }` part is the most important, its what forces the user to get prompted to sign in again instead of automatically being logged in.

```js       
<Button onClick={() => signIn("azure-ad", { callbackUrl: "/dashboard" }, { prompt: "login" })}>
          Sign in with Azure Active Directory 
</Button>
```

**Reply by kaijuh** [reply]
@damain What are you using instead of next-auth?

I created a new discussion because I was having issues with next-auth and multiple google accounts. Take a look and see if you have seen this issue before.

[Google OAuth/Next Auth won't create new users when login in with multiple accounts. ](https://github.com/nextauthjs/next-auth/discussions/11626)

**Reply by damain** [reply]
@kaijuh I am still using next auth (I know other devs who switched to luciaAuth) however, as I added in my comment above I don't think switching is the solution. using the right settings when logging users in and out is the answer to this problem. 

In this issue if the user was currently signed in to the browser it would automatically use the last signed in user and log them in because they are still authenticated in the browser even though they logged out of the app.

**Comment by Gherdty** [comment]
> <Button onClick={() => signIn("azure-ad", { callbackUrl: "/dashboard" }, { prompt: "login" })}>

@damain thank you for your answer! This worked for me for both Azure and Google logins.
